resources/measures/upgrade_hvac_vrf_hr_doas/measure.rb

```diff
@@ -675,16 +727,22 @@ def air_loop_hvac_served_by_district_energy?(air_loop_hvac)
     served_by_district_energy
   end
 
+  # --------------------------------------- #
+  # supporting method
   # check if air loop is served by DOAS
+  # --------------------------------------- #
   def air_loop_hvac_served_by_doas?(air_loop_hvac)
```

Comment:
I am not sure this system sizing method will always work. You might consider also checking for "DOAS" in the name of the airloop.

Reply:
@ChristopherCaradonna I'm definitely not following. could you elaborate?